src/NServiceBus.NHibernate/TimeoutPersisters/NHibernateTimeoutStorage.cs

```diff
@@ -54,10 +56,14 @@ protected override void Setup(FeatureConfigurationContext context)
                 }
             }
 
+            var timeoutsCleanupExecutionInterval = 
+                context.Settings.GetOrDefault<TimeSpan?>("NHibernate.Timeouts.CleanupExecutionInterval") ?? TimeSpan.FromMinutes(2);
```

Comment:
Should we have a public API for overriding the default here? Also, we should document why 2 minutes is a good default (and not 1 minute or 10 minutes). I've seen somebody rightfully recently challenged the "7 day rule" of outbox that we have no scientific data to back with.